### DISCUSSION TITLE
how to find common neighbors in graph

### DISCUSSION THREAD

**Discussion by QuangHDMZ** [JanusGraph/janusgraph#3322] [Q&A]
Hi all, 

I want to predict link between two node using common neighbors. I dont know how to find common neighbors in Janusgraph so I need your help.

Thanks all.

**Comment by vtslab** [comment]
Good question that is probably better suited to Stack Overflow. Could you post the question there and add the [gremlin],  [tinkerpop] and [janusgraph] tags? Please also specify that you envision something like the Common Neighbors algorithm as in [Neo4j Graph Datascience](https://neo4j.com/docs/graph-data-science/current/alpha-algorithms/common-neighbors/).
